README.md

```diff
@@ -60,6 +60,7 @@ Maven plugin for Sonatype Central Portal API ([API Doc](https://central.sonatype
 | central.maxWaitTimePublishing | Maximum wait time in seconds for publishing to complete                                      | 600                                           | 900                                           |
 | central.pollInterval     | Polling interval in seconds when checking deployment status                                       | 5                                             | 10                                            |
 | central.waitForCompletion | If true, wait for complete publishing process. If false, return after validation/publishing starts | false                                          | true                                         |
+| central.showAllDeployments | Lists all deployments available (target: rc-list)                                                 | false                                          | true                                         |
```

Comment:
I suggest `target` -> `goal` (which is more maven-like name)

Reply:
changed